Formula/no-more-secrets.rb

```diff
@@ -0,0 +1,16 @@
+class NoMoreSecrets < Formula
+  desc 'Recreates the SETEC ASTRONOMY effect from "Sneakers"'
```

Comment:
Should probably be reversed, so the single quotes are inside the `desc` element rather than wrapping it.